src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -1,29 +1,60 @@
+import { format } from "date-fns";
+
+import InputTextArea from "../design-system/InputTextArea";
 import PromoBoxWithButton from "../design-system/PromoBoxWithButton";
 import ToggleRow from "./ToggleRow";
+import { Scenario } from "./types";
+
+interface Props {
+  scenario?: Scenario | null;
+  updateScenario: (scenario: Scenario) => void;
+}
+
+const ScenarioSidebar: React.FC<Props> = (props) => {
+  const { scenario, updateScenario } = props;
+  const updatedAtDate = Number(scenario?.updatedAt.toDate());
+
+  const handleScenarioChange = (scenarioChange: object) => {
+    updateScenario(Object.assign({}, scenario, scenarioChange));
+  };
 
-const ScenarioSidebar: React.FC = () => {
   return (
     <div className="flex flex-col w-1/4 mr-24">
       <div className="flex-1 flex flex-col pb-4">
-        <h1 className="text-3xl leading-none">Model 01</h1>
+        <h1 className="text-3xl leading-none">{scenario?.name}</h1>
         <div className="mt-5 mb-5 border-b border-gray-300" />
         <div className="mb-12">
-          <p>
-            This text describes unique qualities for this data model. Taking
-            notes here will be useful when you have multiple data model
-            experiments in your library?
-          </p>
+          <InputTextArea
+            label="Description"
+            value={scenario?.description}
+            placeholder=""
+            onChange={(event) => {
+              // Question - Should we have a Save button to save the description?
```

Comment:
I don't see a save button in the mocks, so I think it should just persist the description change automatically when finished.